libs/editor/WordPressEditor/src/main/java/org/wordpress/android/editor/GutenbergEditorFragment.java

```diff
@@ -98,17 +107,69 @@ private GutenbergContainerFragment getGutenbergContainerFragment() {
         return mRetainedGutenbergContainerFragment;
     }
 
+    public Bundle getTranslations() {
+        Bundle translations = new Bundle();
+        Locale defaultLocale = new Locale("en");
+        Resources currentResources = getResources();
+        Configuration configuration = currentResources.getConfiguration();
+        if (configuration.locale.equals(defaultLocale)) {
+            return translations;
+        }
+
+        DisplayMetrics metrics = new DisplayMetrics();
+        getActivity().getWindowManager().getDefaultDisplay().getMetrics(metrics);
+        configuration.locale = defaultLocale;
```

Comment:
I'm confused by this line... should we assign the _desired_ locale here? Like, the same as the one passed down the the RN app? Instead, this line seems to again assign `en` 🤔

Reply:
Nop this is required to get the default (english) resources. We'll need both the current resource which is whatever the app is set to at the moment and the english ones to generate our `english => [ translated ]` map

Reply:
Aha, right. Yeah, I guess it would be useful if we add some line comments to describe what's going on in the various code paragraphs of this method.